spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConstants.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark.connector.hive;
+
+import com.datastrato.gravitino.catalog.hive.HiveTablePropertiesMetadata;
+import com.datastrato.gravitino.catalog.hive.HiveTablePropertiesMetadata.StorageFormat;
+import com.google.common.annotations.VisibleForTesting;
+
+public class HivePropertiesConstants {
+  public static final String GRAVITINO_HIVE_FORMAT = HiveTablePropertiesMetadata.FORMAT;
+  public static final String GRAVITINO_HIVE_INPUT_FORMAT = HiveTablePropertiesMetadata.INPUT_FORMAT;
+  public static final String GRAVITINO_HIVE_OUTPUT_FORMAT =
+      HiveTablePropertiesMetadata.OUTPUT_FORMAT;
+  public static final String GRAVITINO_HIVE_SERDE_LIB = HiveTablePropertiesMetadata.SERDE_LIB;
+  public static final String GRAVITINO_HIVE_SERDE_PARAMETER_PREFIX =
+      HiveTablePropertiesMetadata.SERDE_PARAMETER_PREFIX;
+
+  public static final String GRAVITINO_HIVE_FORMAT_PARQUET = StorageFormat.PARQUET.toString();
+  public static final String GRAVITINO_HIVE_FORMAT_SEQUENCEFILE =
+      StorageFormat.SEQUENCEFILE.toString();
+  public static final String GRAVITINO_HIVE_FORMAT_ORC = StorageFormat.ORC.toString();
+  public static final String GRAVITINO_HIVE_FORMAT_RCFILE = StorageFormat.RCFILE.toString();
+  public static final String GRAVITINO_HIVE_FORMAT_TEXTFILE = StorageFormat.TEXTFILE.toString();
+  public static final String GRAVITINO_HIVE_FORMAT_AVRO = StorageFormat.AVRO.toString();
+  public static final String GRAVITINO_HIVE_FORMAT_JSON = StorageFormat.JSON.toString();
+  public static final String GRAVITINO_HIVE_FORMAT_CSV = StorageFormat.CSV.toString();
+
+  public static final String SPARK_HIVE_STORED_AS = "hive.stored-as";
+  public static final String SPARK_HIVE_INPUT_FORMAT = "input-format";
+  public static final String SPARK_HIVE_OUTPUT_FORMAT = "output-format";
+  public static final String SPARK_HIVE_SERDE_LIB = "serde-lib";
```

Comment:
Are these configurations defined by Spark or by you?

Reply:
spark use raw string like 'hive.stored-as'